migrations/sql/01_init.up.sql

```diff
@@ -1,7 +1,7 @@
 CREATE TABLE metrics (
   -- Main dimensions
   `queryid` LowCardinality(String) COMMENT 'hash of query fingerprint',
-  `server` LowCardinality(String) COMMENT 'IP or hostname of DB server',
+  `service_name` LowCardinality(String) COMMENT 'Name of service (IP or hostname of DB server by default)',
```

Comment:
I'm not sure that changing previously created migrations is a good practice

Reply:
@BupycHuk,  yes, I agree it's not good. But clickhouse does not allow change index columns. While we are still in beta, and can brake compatibility we need to rename this column to avoid future confusion why field have different names but have same values in dbs, apis, apps.

Reply:
Those days are the last when we can edit existing migrations. With updates coming soon, we should stop doing that.
Let it be the last one.